src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -62,66 +89,66 @@ export const AddressModal: React.FC<Props> = ({
               type="text"
               title="Full Name"
               onChange={formik.handleChange}
-              error={formik.errors.name}
-              value={formik.values.name}
+              error={formik.errors?.name}
+              value={formik.values?.name}
```

Comment:
Are these `?`s needed? Formik is supposed to handle that and return an object for values and errors.

Reply:
Seems fine to remove them